man/portablectl.xml

```diff
@@ -155,6 +155,20 @@
         to be used in case the unit names do not match the image name as described in the <command>attach</command>.</para>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>reattach</command> <replaceable>IMAGE</replaceable> [<replaceable>PREFIX…</replaceable>]</term>
+
+        <listitem><para>Detaches an existing portable service image from the host, and immediately attaches the new
+        specified version. Running units are not stopped during the process. Partial matching, to allow for different
+        versions in the image name, is allowed: only the part before the first <literal>_</literal> character have
+        to match. If the new image doesn't exist, the existing one will not be detached. The parameters follow the
+        same syntax as the <command>attach</command> command.</para></listitem>
```

Comment:
hmm, the prefix handling is the same as for "attach", no? there are no different semantics here? if there are, why? if not, maybe just reference the documentation for "attach" above

Reply:
Yes it's all the same. Shortened to just mention the 'attach' case.